### ISSUE TITLE
should silent join

### ISSUE THREAD

**Issue by jonorthwash** [apertium/phenny#403]
Begiak says `(19:40:36) begiak: Server is up and running on port 1234` when it joins all channels.  It should not.

**Comment by Androbin** [comment]
Yeah, it mistakenly repeats that message, even when the git hook is already started...

**Comment by jonorthwash** [comment]
No, it's still doing it, with a different message now:
```
(21:33:11) begiak [~begiak@apertium.vm.bytemark.co.uk] entered the room.
(21:33:11) begiak: Git hook started on port 1234
```

**Comment by Androbin** [comment]
Fixed in 6a4b9abd5f9a3d2ca7dc1117cf7903f81e2a61b4.
